src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/Review/index.tsx

```diff
@@ -96,11 +95,14 @@ const ContractInteractionReview = ({ onClose, onPrev, tx }: Props): React.ReactE
       onSubmit={submitTx}
       onBack={onPrev}
     >
-      <ModalHeader onClose={onClose} subTitle="2 of 2" title="Contract interaction" />
+      <ModalHeader onClose={onClose} subTitle="Step 2 of 2" title="Contract interaction" />
```

Comment:
Is this consistently named across all flows now?

Reply:
For consistency, you could make a helper function to unify it all, e.g.:

```ts
const getStepTitle = (currentStep: number, totalSteps: number): string => `Step ${currentStep} of ${totalSteps}`
```

Reply:
Thats a clean solution I like that!